src/mission_ui.cpp

```diff
@@ -104,7 +104,7 @@ void mission_ui::draw_mission_ui()
 
         p_impl.last_action = ctxt.handle_input();
 
-        if( p_impl.last_action == "QUIT" ) {
+        if( p_impl.last_action == "QUIT" || !p_impl.get_is_open() ) {
```

Comment:
Make this a separate bug fix please.

Reply:
I agree with you in principle but it should be wholly uncontroversial and I've just learned

![image](https://github.com/CleverRaven/Cataclysm-DDA/assets/84619419/2b9886f2-03d4-4181-89ce-827be607a19a)

...So I'm going to leave it in.